wordpress-mobile/WordPress-FluxC-Android
# ListStore actions should be completed in the bg thread

Fixes #1201 

This PR moves some of the ListStore actions to the background thread.

To test
checkout WPAndroid's `temp/test-list-store-actions-bg-thread` branch which adds strict ThreadPolicy after the PostListFragment is initialized.
1. My Site
2. Blog Posts
3. Scroll/Pull-to-refresh/Change tabs and make sure none of the threading errors in the LogCat is related to the ListStore.

Note:
According to the documentation Coroutine's GlobalScope should be used only in very rare cases. I've created an issue #9527 so we can replace it with a better approach after we explore our options.
